java/google/registry/flows/domain/DomainCreateFlow.java

```diff
@@ -221,12 +222,12 @@ public final EppResponse run() throws EppException {
       verifyPremiumNameIsNotBlocked(targetId, now, clientId);
       verifyNoOpenApplications(now);
       verifyIsGaOrIsSpecialCase(tldState, isAnchorTenant);
-    }
-    String signedMarkId = hasSignedMarks
+      signedMarkId = hasSignedMarks
```

Comment:
Indentation is off here.  Also, what's the purpose of moving the declaration up to line 204 if not using it until down here still?  Is there supposed to be a business logic change here?